src/Microsoft.Health.Dicom.SqlServer/Features/Query/SqlQueryGenerator.cs

```diff
@@ -18,35 +17,26 @@
 
 namespace Microsoft.Health.Dicom.SqlServer.Features.Query
 {
-    internal class SqlQueryGenerator : QueryFilterConditionVisitor
+    internal class SqlQueryGenerator : BaseSqlQueryGenerator
```

Comment:
Seems like we should make this name more specific, like perhaps `QidoSqlQueryGenerator`

Reply:
Keeping the name same to avoid touching lot of files